index.html

```
@@ -1995,9 +2030,14 @@ <h3 class="font-bold mb-2">Join Discord</h3>
       <div class="proposal-header-title">
         <h2 id="pwOrgName">Organization Proposal</h2>
-          <label class="proposal-autosave">
+          <label for="pwAutosaveToggle" class="proposal-autosave">
             <input type="checkbox" id="pwAutosaveToggle" aria-describedby="pwPrivacyNote" />
             Enable autosave
+          </label>
+          <p id="pwPrivacyNote">
+            Drafts are stored only on this device when autosave is enabled.
+          </p>          
+            Enable autosave
             <p id="pwPrivacyNote">Drafts are stored only on this device when autosave is enabled.</p>
```

HTMLHint (1.9.2)

[error] The id value [ pwPrivacyNote ] must be unique.

(id-unique)